content/integrations/faq/haproxy-multi-process.md

```diff
@@ -0,0 +1,27 @@
+---
+title: HAProxy in multi-process mode
+kind: faq
+---
+
+When using the HAProxy in multi process mode, each process has its own memory area and therefore has its own statistics.
+
+{{< img src="integrations/faq/haproxy_config_multi_process.png" alt="HAProxy config multi process" responsive="true" popup="true" style="width:30%;">}}
+
+This implies that each process needs to have a dedicated socket or endpoint for giving access to its statistics.  
+Accordingly, in the Datadog configuration file for HAProxy, **each socket or endpoint has to be declared as an instance**.
+
+{{< img src="integrations/faq/haproxy_multi_process_agent_conf.png" alt="HAProxy multiprocess configuration" responsive="true" popup="true" style="width:50%;">}}
+
+
+Otherwise they share the same `/haproxy_stats` endpoint, and getting the HAProxy statistics from it shows the stats only for the process assigned to the current request.
+
+{{< img src="integrations/faq/haproxy_stats_1.png" alt="HAProxy stats 1" responsive="true" popup="true" style="width:85%;" >}}
+
+Refreshing the page in your browser shows the stats from a different process than previously:
+
+{{< img src="integrations/faq/haproxy_stats_2.png" alt=" HAProxy stats 2" responsive="true" popup="true" style="width:85%;" >}}
+
+If your HAproxy integration is no well configured, i.e., you may notice:
```

Comment:
Should this be: `If your HAproxy integration is not well configured, you may notice:` instead of `If your HAproxy integration is no well configured, i.e., you may notice:`?

Reply:
yup, you are right!